doc/nrf/ug_zigbee_configuring_components.rst

```diff
@@ -0,0 +1,31 @@
+﻿.. _ug_zigbee_configuring_components:
+
+Configuring Zigbee components in |NCS|
```

Comment:
I think we need to decide on a single name for this. So far I see several
1) Zigbee libraries
2) Zigbee helper functions
3) Zigbee components

I think the best name is either 1) or "Zigbee Application Components".

Reply:
Let's stick to 1) for now.